Frameworks/CoreGraphics/CGFont.mm

```diff
@@ -40,6 +41,9 @@
     CFRuntimeBase _base;
     ComPtr<IDWriteFontFace> _dwriteFontFace;
 
+    // Contains a value when created via Data Provider, null o/w
+    woc::unique_cf<CFDataRef> _data;
```

Comment:
makes sense, can you jot a comment down about the tradeoff then? i agree that this is preferable for the time being, but keeping a ttf file entirely in memory is still less than desirable.